src/PdfReader/index.tsx

```diff
@@ -74,30 +101,45 @@ export default function usePdfReader(args: ReaderArguments): ReaderReturn {
   const setColorMode = React.useCallback(async () => {
     console.log('unimplemented');
   }, []);
-  const setScroll = React.useCallback(async () => {
-    console.log('unimplemented');
-  }, []);
+
+  const setScroll = React.useCallback(
+    async (val: 'scrolling' | 'paginated') => {
+      const isScrolling = val === 'scrolling';
+      dispatch({ type: 'SET_SCROLL', isScrolling });
+    },
+    []
+  );
 
   // this format is inactive, return null
   if (!webpubManifestUrl || !manifest) return null;
 
-  const isLoading = true;
-
   // we are initializing the reader
   if (isLoading) {
     return {
       isLoading: true,
-      content: <PDFContent />,
+      content: <>PDF is loading</>,
       manifest: null,
       navigator: null,
       state: null,
     };
   }
 
+  function onDocumentLoadSuccess({ numPages }: { numPages: number }) {
+    setNumPages(numPages);
+  }
```

Comment:
I would make a reducer action called `DOCUMENT_LOAD_SUCCESS` and and pass anything we need from here to that to be set inside the `state` variable.

Reply:
The presence of "state.data" is a better indicator of load success than anything else - so I removed it.  Let me know if this all makes sense anyway.